src/super_gradients/training/pre_launch_callbacks/pre_launch_callbacks.py

```diff
@@ -109,6 +109,7 @@ def __call__(self, cfg: DictConfig) -> DictConfig:
         tmp_cfg.training_hyperparamsbatch_accumulate = 1
         tmp_cfg.training_hyperparamsmax_train_batches = self.num_forward_passes
         tmp_cfg.training_hyperparamsrun_validation_freq = 2
+        tmp_cfg.training_hyperparamsrun_test_freq = 2
```

Comment:
I feel we have a typo error in this whole section:
`tmp_cfg.training_hyperparamsrun_test_freq = 2`

There is a missing dot after `tmp_cfg.training_hyperparams`:
`tmp_cfg.training_hyperparams.run_test_freq = 2`

Reply:
@shaydeci how that is supposed to be working in principle if we have that many typos?

Reply:
Completely missed that one.
Please fox @Louis-Dupont

Reply:
Fixed to `tmp_cfg.training_hyperparams.run_test_freq = 2`